### PR TITLE
Update description related to Workflow Catalog Query Language

### PR DESCRIPTION
**Pull request by lpellegr**

### CODE REVIEW

**Comment by lpellegr** [comment]
@ow2-proactive/developers Could two people please review the changes?

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -1145,7 +1145,7 @@ It is also possible to list all Workflows inside a particular Bucket using its `
 
     GET http://localhost:8080/workflow-catalog/buckets/{bucket_id}/workflows
 
-Then we can fetch a particular Workflow based on its `workflow_id`:
+Then, we can fetch a particular Workflow based on its `workflow_id`:
 
     GET http://localhost:8080/workflow-catalog/buckets/{bucket_id}/workflows/{workflow_id}
```
Comment (cdiop) [inline src/docs/user/ProActiveUserGuide.adoc]: May be replace localhost by CATALOG_ADDRESS

**Comment by cdiop** [comment]
Oki for me/

**Comment by lpellegr** [comment]
Thank you Codé!

Could one more person please have a look? 

@paraita ;) ?

**Comment by paraita** [comment]
Looks good for me :+1:

**Pull request merged**